### PR TITLE
EIP-1559 intro to past tense

### PR DESCRIPTION
**Pull request by ruleking**
London upgrade text should be in past tense.

### CODE REVIEW

File: src/content/developers/docs/gas/index.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ To execute a transaction on the network users are able to specify a maximum limi
...
 
-The implementation of [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md) in the London Upgrade makes the transaction fee mechanism more complex than the current gas price auction, but it has the advantage of making gas fees more predictable, resulting in a more efficient transaction fee market. Users can submit transactions with a `maxFeePerGas` corresponding to how much they are willing to pay for the transaction to be executing, knowing that they will not pay more than the market price for gas (`baseFeePerGas`), and get any extra, minus their tip, refunded.
+The implementation of [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md) in the London Upgrade did make the transaction fee mechanism more complex than the previous gas price auction, but it has the advantage of making gas fees more predictable, resulting in a more efficient transaction fee market. Users can submit transactions with a `maxFeePerGas` corresponding to how much they are willing to pay for the transaction to be executing, knowing that they will not pay more than the market price for gas (`baseFeePerGas`), and get any extra, minus their tip, refunded.
```
Comment (minimalsm) [inline src/content/developers/docs/gas/index.md:109]:
```suggestion
The implementation of [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md) in the London Upgrade made the transaction fee mechanism more complex than the previous gas price auction, but it has the advantage of making gas fees more predictable, resulting in a more efficient transaction fee market. Users can submit transactions with a `maxFeePerGas` corresponding to how much they are willing to pay for the transaction to be executing, knowing that they will not pay more than the market price for gas (`baseFeePerGas`), and get any extra, minus their tip, refunded.
```

Grammatically, I think this makes more sense.

**Review by wackerow — APPROVED** [review]
@ruleking thanks! 👍🏻

**Comment by wackerow** [comment]
@all-contributors please add @ruleking for bug patch

**Comment by minimalsm** [comment]
Thanks for spotting and fixing this @ruleking  ✌️.

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳!

**Pull request merged**